src/main.rs

```diff
@@ -1,53 +0,0 @@
-use anyhow::Result;
```

Comment:
Since these files were removed, can the `logging.rs` in the same directory be removed?